### ISSUE TITLE
Tried accessing nonexisting field (first_name) on node type (Page)

### ISSUE THREAD

**Issue by damienpicca** [botman/botman#525]
- BotMan Version: 2.0
- PHP Version: 7.0.3
- Messaging Service(s): Messenger
- Cache Driver:

### Description:
Hello, I try to setup my webhook but it is impossible to get user id in order to initiate a message.
Here is the portion of code 

```
            // Apply global "received" middleware
	    $botman->middleware->received($apiAi);

	    // Apply matching middleware per hears command
	    $botman->hears('trackflight', function (BotMan $bot) {
                     $user = $bot->getUser();
	             $id = $user->getId();

                    $extras = $bot->getMessage()->getExtras();
                    $apiReply = $extras['apiReply'];
                    $bot->reply($apiReply);
		})->middleware($apiAi);


	    $params = array();
	    $content = $request->get("request");
	    if (!empty($content))
	    {
	        $params = json_decode($content, true); 

	        $botman->say('message test ', $id);

	    }

	    $botman->listen();
```

When API.AI sends request to webhook (thus when $content isn't empty) an error is returned : *Error sending payload: (#100) Tried accessing nonexisting field (first_name) on node type (Page) (500 Internal Server Error)*

I thank you and I hope that you can enlighten me :) 
### Steps To Reproduce:

**Comment by johntiror** [comment]
I had this issue, but it was not related to API.AI
I solved subscribing my bot only to "message" and "message_postbacks"
Let me know if this solve your issue.

**Comment by mpociot** [comment]
Fixed with Facebook Driver 1.0.3
